src/libmscore/lyrics.h

```diff
@@ -82,7 +82,7 @@ class Lyrics final : public TextBase
     QString subtypeName() const override { return QObject::tr("Verse %1").arg(_no + 1); }
     void setNo(int n) { _no = n; }
     int no() const { return _no; }
-    bool isEven() const { return _no % 1; }
+    bool isEven() const { return !(_no % 2); }
```

Comment:
Outch! That Mega Hertz!

Reply:
This bug is in 3.x too BTW

Reply:
See #7628

Reply:
Might (!?) resolve:

* https://musescore.org/en/node/9304
* https://musescore.org/en/node/285495
* https://musescore.org/en/node/288604

Reply:
That fix seems wrong though, `_no` is zero-based. and `0 % 2` is `0` and `!0` is true, but verse 1 is odd. So just drop that `!` or add a '  + 1` to `_no` (prior to the modulo operation).

Reply:
Another possible fix is to just replace the `%` with a `&`, that's a method used in other places.

Reply:
I don't understand why the fix is wrong, the function is `isEven()` so if `_no` is `0` it should return `true`, right?

Reply:
I don't understand why the fix is wrong, the function is `isEven()` so if `_no` is `0` it should return `true`, right?

Reply:
No0 if `_no` is 0, that is verse 1 and 1 is odd

Reply:
No, if `_no` is 0, that is verse 1 and 1 is odd. So for 0 and 2 that should return false, for 1 and 3 true. Yes, that is quite off (pun intended)

Reply:
Hence my proposed fix with changing the `%` to `&`.

Reply:
LMAO, than my first pull request was [correct](https://github.com/musescore/MuseScore/pull/7626#discussion_r588095924)

Reply:
Oh I understand now. Yes, `_no & 1` is correct. And I believe it would've been the intention of the person who wrote this part of the code.

Reply:
But I'd suggest adding a comment explaining why it should be the opposite of the apparent meaning.

Reply:
`// _no is 0-based`